### DISCUSSION TITLE
2022-02-23 Weekly Meeting 44 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#183] [Weekly Meetings]
Weekly Meeting 44 is happening on Wednesday Feb 22nd, 2022 from 9:00 AM PDT - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Announcements and Updates
• Interoperator starts on Friday at 9am PT
• This item Proposed changes to charter, and touches on some potentially sensitive considerations given the changes are related to revealing identity for validity. We gave it a week to review: Proposal: Revise Specifications Section of Charter #180

Working Group Updates:
-  OMI Website update
   - Interoperator group update: "friends of OMI section"
- OMI-Chan group
  - https://github.com/omigroup/omi-chan/discussions/25 
- WASM group
- gLTF group
- any others?

Housekeeping:
Any?

Open Floor!
 
Please leave any additional agenda items below in the comments. Thanks!

**Comment by LightLodges** [comment]
I'd like to propose use of the MIRO version of the Platform Design Toolkit for stakeholder mapping, incentives alignment with engagement mapping and the background community strategy work we'll need to do the next stage of work here at OMI. Using MIRO, this can be community facilitated over 1 long session or 2 shorter meeting times in March.

https://miro.com/miroverse/platform-design-toolkit/  -- not all of this canvas may be necessary for us as we are not creating a single platform, but a community of this type does exist to support each other and the whole as a network with related dynamics.

This is the next stage of the strategy roadmap & would also serve to outline aspects of Interoperators in terms of language structures/incentives/roles for participation with this community down the line.

We need to make some big picture decisions ahead for the long term resiliency of the open metaverse as well as for the support of this community.

**Comment by mrmetaverse** [comment]
###### note: Today is my birthday, so I _might_ be AFK during our meeting. 
- new item for review, per Evo: https://github.com/omigroup/omigroup/issues/184
- backlog refinement working session 2/22/22 [notes](https://hackmd.io/@mrmetaverse/omi-backlog-022222)
  - We groomed the entire holistic backlog. Everything 'done' is actually in the done column. All To Do and In Progress are up to date. Left comments on various tickets to un-block or update the status. Pulled in what made sense from the other working group boards into the holistic board. Will have next public session next month (4th Tuesday of the month). 
- Interoperator starts on Friday at 9am PT 
    - This is the Interoperators Group. The group will kick off this Friday 2/25/22 at 17:00 UTC (9am PT) 
:calendar: add this event to your calendar [using google cal](https://calendar.google.com/event?action=TEMPLATE&tmeid=NzNla2d1MDZzNnYybWNhZDZsN3ZxaDdyYWNfMjAyMjAyMjVUMTcwMDAwWiBjXzBkcHVnZjlmODNlcTRxZWttYjZvbXFiam1nQGc&tmsrc=c_0dpugf9f83eq4qekmb6omqbjmg%40group.calendar.google.com&scp=ALL)
- This item Proposed changes to charter, and touches on some potentially sensitive considerations given the changes are related to revealing identity for validity. We gave it a week to review: https://github.com/omigroup/omigroup/pull/180
